galaxy_ng/tests/integration/api/test_artifact_upload.py

```diff
@@ -485,7 +496,7 @@ def test_api_publish_ignore_files_logged(ansible_config, upload_artifact):
     assert len(ignorefile) == 1  # found ignorefile log message
 
 
-@pytest.mark.cloud_only
+@pytest.mark.deployment_cloud
```

Comment:
updating the marks in run_tests.sh may get the ephemeral CI pr_check.sh to pass https://github.com/ansible/galaxy_ng/blob/95710a7ad56d69f0f931907bee95e50f17fe2c37/dev/ephemeral/run_tests.sh#L15